Packs/CortexXDR/ReleaseNotes/4_9_4.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+##### Cortex XDR - True Positive Incident Handling
+- Fixed an issue of ***replyOptions*** not being aligned with ***nexttasks*** according to new validations.
```

Comment:
I think this gives more clarity for the end users
```suggestion
- Fixed an issue where the reply options given by the conditional task were not aligned with the actual tasks of the playbook.
```

Reply:
https://github.com/demisto/content/pull/22451/commits/0ea6e5a6a38f4df5ae563df2b2fffcab9e208baa